docs/source/refs/reference_architecture/index.rst

```diff
@@ -0,0 +1,392 @@
+    :align: center
+    :alt: Isaac Lab Reference Architecture
+
+
+|
+
+The reference architecture for Isaac Lab comprises the following components:
+
+1. **Asset Input**
+2. **Configuration - Assets & Scene**
+3. **Robot Learning Task Design**
+4. **Register with Gymnasium**
+5. **Environment Wrapping**
+6. **Run Training**
+
+  * Single GPU Training
+  * Multi-GPU Training
+  * Multi-Node Training
+  * Cloud-based Training
+
+7. **Run Testing**
+
+
+**Components**
+-----------------
+In this section, we will briefly discuss the individual blocks for creating a
+sample reference application in Isaac Lab.
+
+**Component 1 - Asset Input**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Isaac Lab accepts URDF or USD files for assets. The first step to training using Isaac Lab is to
+have the USD file of your asset and the USD or URDF file of your robot. This can be achieved in
+the following ways:
+
+
+1. Design your assets or robot in Isaac Sim and export the USD file.
+
+2. Design your assets or robot in any software of your choice and export it to USD using Isaac Sim converters.
+
+  Isaac Sim supports the different converters/importers to USD such as the `CAD Converter`_, `URDF Importer`_, `MJCF Importer`_, `Onshape Importer`_, etc.
+  More details are found in the Importing Assets section in the `Isaac Sim Reference Architecture`_.
+
+3. If you already have the URDF file of your robot, you do not need to convert to USD as Isaac Lab takes URDF.
+
+
+**Component 2 -  Configuration (Assets and Scene)**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Asset Configuration**
+
+Given that you have the asset file for your robot and other assets such as environment objects based on the task, the next step is to import them into Isaac Lab. Isaac Lab uses asset configuration classes to spawn various objects (or prims) into the scene using Python. The first step is to write a
+configuration class to define the properties for the assets needed to complete the task. For example,
+a simple go-to-goal task for a mobile robot will include the robot asset, an object like cubes to
+signify the goal pose visually, lights, ground plane, etc. Isaac Lab understands these assets using
+the configuration classes. Isaac Lab provides various sim-ready assets such as. physically accurate
+3D objects that encompass accurate physical properties and behavior. It also provides connected
+data streams to represent the real world in simulated digital worlds
+such as `robots <https://github.com/isaac-sim/IsaacLab/tree/main/source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets>`__
+like Anyrobotics Anymal, Unitree H1 Humanoid, etc. as well as
+`sensors <https://github.com/isaac-sim/IsaacLab/tree/main/source/extensions/omni.isaac.lab/omni/isaac/lab/sensors>`__,
+`shapes <https://github.com/isaac-sim/IsaacLab/tree/main/source/extensions/omni.isaac.lab/omni/isaac/lab/sim/spawners/shapes>`__,
+lights, and more. We provide their asset file and configuration classes.
```

Comment:
"We provide their asset file and configuration classes.
Users can also define their own assets using the configuration classes." - it's not too clear what does "we provide their asset file and configuration classes" mean. could we clarify the language a bit more?

Reply:
Rephrased:

We provide these assets configuration classes. Users can also define their own assets using the configuration classes.